src/lease/mod.rs

```diff
@@ -1,75 +1,13 @@
 //! Leases are a mechanism for detecting client liveness. The cluster grants leases with a time-to-live. A lease expires if the etcd cluster does not receive a keepAlive within a given TTL period.
-//!
-//! # Examples
```

Comment:
please keep the example

Reply:
The example is written by tokio, I can rewrite them by smol later.